server/core/src/main/java/dev/slimevr/tracking/processor/skeleton/HumanSkeleton.kt

```diff
@@ -519,6 +524,72 @@ class HumanSkeleton(
 		if (isTrackingRightArmFromController) rightHandTrackerBone.update()
 	}
 
+	private fun updateTrackerYawCorrections() {
+		val trackersToAlign = listOf(
+			headTracker,
+			upperChestTracker,
+			chestTracker,
+			waistTracker,
+			hipTracker
+		);
+
+		var parentTracker: Tracker? = null;
+		for (tracker in trackersToAlign) {
+			if (tracker == null) {
+				continue;
+			}
+
+			if (parentTracker == null) {
+				parentTracker = tracker;
+				continue;
+			}
+
```

Comment:
Check if the tracker `isImu`. If it's not, don't adjust for drift.

Reply:
Addressed by adding `isImu` check before updating the yaw. I chose to allow an IMU tracker to be adjusted to match a non-IMU parent tracker.